rgw.conf.sample

```diff
@@ -0,0 +1,7 @@
+{
+  "install_path": "/opt/seagate/cortx",
+  "log_dir": "/var/log",
```

Comment:
Remove log_dir and log_level. These are not required as we dont have running service.

Reply:
shouldn't we capture log configs and apply in ceph.conf later ?